src/app/zap-templates/zcl/data-model/chip/content-launch-cluster.xml

```diff
@@ -29,72 +29,66 @@ limitations under the License.
     <attribute side="server" code="0x0000" define="CONTENT_LAUNCHER_ACCEPT_HEADER" type="ARRAY" entryType="CHAR_STRING" length="254" writable="false" optional="false">accept header list</attribute>
     <attribute side="server" code="0x0001" define="CONTENT_LAUNCHER_SUPPORTED_STREAMING_PROTOCOLS" type="BITMAP32" writable="true" optional="false">supported streaming protocols</attribute>
 
-    <command source="client" code="0x00" name="LaunchContent" optional="false">
+    <command source="client" code="0x00" name="LaunchContentRequest" response="LaunchResponse" optional="false">
```

Comment:
Please file a spec issue about it being inconsistent about "Launch" vs "Launch Response".